DataFormats/HcalDigi/interface/HcalUMNioDigi.h

```diff
@@ -0,0 +1,47 @@
+#ifndef DATAFORMATS_HCALDIGI_HCALUMNIODIGI_H
+#define DATAFORMATS_HCALDIGI_HCALUMNIODIGI_H 1
+
+#include <vector>
+#include <stdint.h>
```

Comment:
This would run counter to most existing CMS code which needs explicit sizes in NOT requiring the "std::uint32_t" and using "uint32_t" directly.  The standards are very confused on this issue in any case, with different interpretations in different places -- mostly because people want these in the global namespace...